Scripts/ComToTcpServer.py

```diff
@@ -27,8 +27,10 @@
                     help="The number of the COM device to connect to. E.G 'COM5'")
 parser.add_argument("-b", "--baudrate", type=int, default=115200,
                     help="The baudrate of the serial port.")
-parser.add_argument("-v", "--verbose", action="store_true",
-                    help="Enabled verbose prints.")
+parser.add_argument("-o", "--output", action="store_true",

```

Comment:
Could we also have a `-l <log_file>` or something option, since the terminal will roll over relatively quickly? If would be nice if both `-o` and `-l` could work at the same time, so you could see it in real time and have it write a log for reference and whatever you missed. But I'll defer to how much work you want to put into it :)

Reply:
@os-d, that already exists above and they should work at the same time